### PR TITLE
[MRG+1] MNT: cleanup whitespace around @_preprocess decorator

### PR DESCRIPTION
**Pull request by phobson**

### CODE REVIEW

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -1082,9 +1081,9 @@ def vlines(self, x, ymin, ymax, colors='k', linestyles='solid',
         return lines
 
     @_preprocess_data(replace_names=["positions", "lineoffsets",
-                                        "linelengths", "linewidths",
-                                        "colors", "linestyles"],
-                         label_namer=None)
+                                  "linelengths", "linewidths",
```
Comment (anntzer) [inline lib/matplotlib/axes/_axes.py]: still wrong.

**Comment by dstansby** [comment]
I thought there was a general policy of no PEP8-only changes? (although personally I'm happy with them)

**Comment by NelleV** [comment]
I don't know if there is an official policy, but I am concerned that this will create a lot of conflicts with opened pull requests. pep8-patches are a pain to rebase against.

**Comment by phobson** [comment]
@NelleV that's an excellent point. happy to toss this out

File: lib/matplotlib/axes/_axes.py [hunk 2]
```diff
@@ -3807,9 +3806,9 @@ def dopatch(xs, ys, **kwargs):
                     medians=medians, fliers=fliers, means=means)
 
     @_preprocess_data(replace_names=["x", "y", "s", "linewidths",
-                                        "edgecolors", "c", 'facecolor',
-                                        'facecolors', 'color'],
-                         label_namer="y")
+                                     "edgecolors", "c", 'facecolor',
+                                     'facecolors', 'color'],
```
Comment (QuLogic) [inline lib/matplotlib/axes/_axes.py]: Can you make these use consistent quotes?

**Comment by NelleV** [comment]
@phobson You'll need to rebase.

**Comment by phobson** [comment]
@NelleV thanks for the heads up. I found some sketchy wifi and rebased.

**Pull request merged**